frontend/src/views/PageNotFound.vue

```diff
@@ -16,53 +16,39 @@
         </v-btn>
       </template>
 
-      <v-row>
-        <v-col
-          offset="0"
-          offset-sm="1"
-          offset-md="2"
-          offset-lg="2"
-          offset-xl="2"
-          cols="12"
-          sm="10"
-          md="8"
-          lg="8"
-          xl="8"
+      <v-card-text class="justify-center">
+        <p id="error-title" class="font-weight-bold text-center px-8 text-h4">
+          {{ $tc('views.pageNotFound.detail') }}
+        </p>
+        <v-img contain class="error-image" src="/public/fourZeroFour.svg"></v-img>
```

Comment:
```suggestion
      <v-card-text class="text-center">
        <img class="error-image" src="@/assets/fourZeroFour.svg">
        <p id="error-title" class="font-weight-bold px-8">
          {{ $tc('views.pageNotFound.detail') }}
        </p>
```

I think if you do it like this, then you can remove it from vuetify.js and the public folder :)
(I would also remove the text-h4 as it makes the paragraph huge on mobile. I think it is enough if it's just a bold paragraph below the image.)